components/tools/ToolDashboard.js

```diff
@@ -5,15 +5,18 @@ import FilterIcon from '../icons/Filter';
 import SearchIcon from '../icons/Search';
 import ToolsList from './ToolsList';
 import Filters from './Filters';
+import Button from '../buttons/Button';
+import Cross from '../icons/Cross';
```

Comment:
You have imported this icon but not added it to the repository. Hence, build is failing in netlify.

Reply:
@akshatnema, I have added the cross icon file now.
You can review now.

Reply:
Yeah. I saw that. Thanks for reminding me. I have asked for review from the designers but there aren't any responses yet. You can see the conversation [here](https://asyncapi.slack.com/archives/C02JW8DA0DC/p1674888341022169). Can you please make the above button as same demonstrated in the video?

Reply:
Thanks :+1:

Reply:
Thanks :+1:

Reply:
Hi @akshatnema, Apologies for the late response.
I have updated the Clear Filters, as demonstrated in the video.

Have a look!